lib/gringotts/gateways/cams.ex

```diff
@@ -1,36 +1,49 @@
 defmodule Gringotts.Gateways.Cams do
   @moduledoc ~S"""
-    An API client for the [CAMS](https://www.centralams.com/) gateway.
-    For referance you can test gateway operations [CAMS API SANDBOX] (https://secure.centralams.com).
+    A module for working with the Cams payment gateway.
 
-    Test it using test crediantials username:***testintegrationc***, password:***password9***
+    An API client for the [CAMS](https://www.centralams.com/) gateway.
+    For referance you can test gateway operations [CAMS API TEST MODE](https://secure.centralams.com).
+    Test it using test crediantials **username:** `testintegrationc`, **password:** `password9`
 
     The following features of CAMS are implemented:
 
-    * **Purchase**  In  `purchase/3`
+    | Action                       | Method        |
+    | ------                       | ------        |
+    | Authorize                    | `authorize/3` |
+    | Capture                      | `capture/3`   |
+    | Purchase                     | `purchase/3`  |
+    | Refund                       | `refund/3`    |
+    | Cancel                       | `void/2`      |
+
 
-    * **Authorize** In  `authorize/3`
+  ## The `opts` argument
 
-    * **Capture**   In  `capture/3`
+    Most `Gringotts` API calls accept an optional `Keyword` list `opts` to supply
+    optional arguments for transactions with the Cams gateway. The following keys
+    are supported:
+    
+    | Key                 | Remark | Status          |
+    | ----                | ---    | ----            |
+    | `billing_address`   |        | Not implemented |
+    | `address`           |     	 | Not implemented |
```

Comment:
@gopalshimpi the formatting is totally out of places. please make sure to review your PR in github also once you push your changes.